src/js/modules/infragistics.ui.combo.js

```diff
@@ -1965,9 +1965,31 @@
 			if ($.ig && $.ig.formatter) {
 				if (this.options.format === "auto" &&
 					($.type(item) === "date" || $.type(item) === "number")) {
-					item = $.ig.formatter(item, null, null);
+					item = $.ig.formatter(
```

Comment:
It won't be breaking change if we test for the type of the first parameter and split the logic in the formatter based on that check.